plugins/catalog-backend/optic.yml

```diff
@@ -0,0 +1,15 @@
+ruleset:
+  - breaking-changes
+capture:
+  src/schema/openapi.yaml:
+    # 🔧 Runnable example with simple get requests.
+    # Run with "PORT=3000 optic capture src/schema/openapi.yaml --update interactive" in 'plugins/catalog-backend'
+    # You can change the server and the 'requests' section to experiment
+    server:
+      # This will not be used by 'backstage-repo-tools schema openapi test', but may be useful for interactive updates.
+      url: http://localhost:3000
+    requests:
+      # ℹ️ Requests should be sent to the Optic proxy, the address of which is injected into 'run.command's env as OPTIC_PROXY (or the value of 'run.proxy_variable').
+      run:
+        # 🔧 Specify a command that will generate traffic
+        command: yarn backstage-cli package test
```

Comment:
Regarding running this with `CI=1`, I'm thinking it's best if we tweak this to work in isolation, it's a standalone request generation command after all. While at it we could slim it down a bit too to avoid running all the tests.

Something like this:

```sh
yarn backstage-cli package test --no-watch src/service/createRouter.test.ts
```

Reply:
Makes sense, updated!